src/main/java/net/atos/client/zgw/shared/ZGWApiService.java

```diff
@@ -38,11 +38,7 @@
 import net.atos.client.zgw.zrc.model.ZaakInformatieobject;
 import net.atos.client.zgw.zrc.model.generated.Resultaat;
 import net.atos.client.zgw.ztc.ZtcClientService;
-import net.atos.client.zgw.ztc.model.generated.BrondatumArchiefprocedure;
-import net.atos.client.zgw.ztc.model.generated.ResultaatType;
-import net.atos.client.zgw.ztc.model.generated.RolType;
-import net.atos.client.zgw.ztc.model.generated.StatusType;
-import net.atos.client.zgw.ztc.model.generated.ZaakType;
+import net.atos.client.zgw.ztc.model.generated.*;
```

Comment:
Do we want to have wildcard imports here? I guess if this is generated we don't really mind

Reply:
No, we don't. Will fix. This is not generated code.

Reply:
Fixed now

Reply:
Almost.. Bit annoying. Spotless does not seem to pick up these things.